wavefront/server/modules/plugins_module/plugins_module/utils/helper.py

```
@@ -53,6 +53,23 @@ def generate_cache_key(
     return f'dynamic_query:{hash_digest}'
 
 
+def generate_export_filename_hash(
+    filter: str = None,
```

Ruff (0.15.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)